tasks/eva_3417/src/main/groovy/eva3417/HashCollisionWithDifferentRSRemediation.groovy

```diff
@@ -0,0 +1,573 @@
+                //both the RS are valid
+                return CATEGORY.BOTH_SVE_HAS_RS_BOTH_VALID
+            } else if (!rsInFileValid || !rsInDBValid) {
+                // one RS is valid while the other is Invalid
+                return CATEGORY.BOTH_SVE_HAS_RS_ONE_VALID
+            }
+        } else if (rsInFile != null || rsInDB != null) {
+            // one of the SVE contains RS
+            if (!rsInFileValid && !rsInDBValid) {
+                // one of the SVE contains RS but that RS is not valid
+                return CATEGORY.ONE_SVE_HAS_RS_NONE_VALID
+            } else if (rsInFileValid || rsInDBValid) {
+                // one of the SVE contains a valid RS
+                return CATEGORY.ONE_SVE_HAS_RS_ONE_VALID
+            }
+        }
+    }
+
+    List<SubmittedVariantEntity> findSVEAccessionInOrgAssembly(String asm, Long acc) {
+        List<SubmittedVariantEntity> dbSVEList = [sveClass, dbsnpSveClass]
+                .collectMany(ssClass -> dbEnv.mongoTemplate.find(query(where("seq").is(asm).and("accession").is(acc)), ssClass))
+                .flatten()
+        return dbSVEList
+    }
+
+    SubmittedVariantEntity getRemediatedSVE(SubmittedVariantEntity submittedVariantEntity) {
+        SubmittedVariant sveModel = submittedVariantEntity.getModel()
+        sveModel.setReferenceAllele(submittedVariantEntity.getReferenceAllele().toUpperCase())
+        sveModel.setAlternateAllele(submittedVariantEntity.getAlternateAllele().toUpperCase())
+        return new SubmittedVariantEntity(submittedVariantEntity.getAccession(),
+                getSVENewHash(submittedVariantEntity), sveModel, submittedVariantEntity.getVersion())
+    }
+
+    SubmittedVariantOperationEntity getSVOEForMergeOperation(SubmittedVariantEntity submittedVariantEntity, Long accessionMergeInto) {
+        // remediate sve if it contains lowercase nucleotide before putting in svoe
+        SubmittedVariantEntity sve = getRemediatedSVE(submittedVariantEntity)
+
+        // create merge operation
+        SubmittedVariantOperationEntity svoe = new SubmittedVariantOperationEntity()
+        svoe.fill(EventType.MERGED, sve.getAccession(), accessionMergeInto,
+                "After fixing lowercase nucleotide issue, variant merged due to duplicate hash",
+                Collections.singletonList(new SubmittedVariantInactiveEntity(sve)))
+        svoe.setId("EVA3417_MERGED_" + sve.getAccession() + "_INTO_" + accessionMergeInto + "_HASH_" + sve.getHashedMessage())
+
+        return svoe
+    }
+
+
+    void remediateCategoryAtMostOneValidRS() {
+        String[] affectedAsmList = [
+                "GCA_000002285.2", "GCA_000002315.3", "GCA_000003055.5", "GCA_011100555.1", "GCA_000002035.3",
+                "GCA_000001635.6", "GCA_000349105.1", "GCA_016772045.1", "GCA_000003025.6", "GCA_000349185.1",
+                "GCA_000247795.2", "GCA_000473445.2",
+                "GCA_001433935.1"
+        ]
```

Comment:
note that to run this for `GCA_002263795.2` I'll need to modify this list, will do it directly in the version deployed in the cluster